apple/testing/default_runner/ios_test_runner.template.sh

```diff
@@ -76,14 +77,14 @@ if [[ -n "${LAUNCH_OPTIONS_JSON_STR}" ]]; then
   LAUNCH_OPTIONS_JSON_STR="{${LAUNCH_OPTIONS_JSON_STR}}"
   LAUNCH_OPTIONS_JSON_PATH="${TMP_DIR}/launch_options.json"
   echo "${LAUNCH_OPTIONS_JSON_STR}" > "${LAUNCH_OPTIONS_JSON_PATH}"
-  runner_flags+=(--launch_options_json_path="${LAUNCH_OPTIONS_JSON_PATH}")
+  runner_flags+=("--launch_options_json_path=${LAUNCH_OPTIONS_JSON_PATH}")
 fi
 
 cmd=("%(testrunner_binary)s"
   "${runner_flags[@]}"
   simulator_test
-  --device_type="%(device_type)s"
-  --os_version="%(os_version)s"
+  "--device_type=%(device_type)s"
```

Comment:
Do we need quotes around the values for device type and os version?

Reply:
Since the whole argument will be passed as `'--device_type=iPhone X'` it should be fine with spaces